### PR TITLE
Extend IPUWorkflowConfig actor with OS release and architecture

### PR DESCRIPTION
**Pull request by vojtechsokol**

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/library.py [hunk 1]
```diff
@@ -12,3 +13,26 @@ def get_env_vars():
     available after reboot.
     """
     return [EnvVar(name=k, value=v) for (k, v) in os.environ.items() if k.startswith('LEAPP_') and k not in ENV_IGNORE]
+
+
+def get_os_release(path):
+    """Retrieve data about System OS release from provided file."""
```
Comment (Rezney) [inline repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/library.py:19]: Docstrings should be like this in our project; from what I have learnt recently:

"""
text
"""

**Review by Rezney — CHANGES_REQUESTED** [review]
We should probably also refactor "OSReleaseCollector" and "CheckOSRelease" to not gather it twice. And please also add tests.

File: repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/library.py (hunk 1 above)
Comment (vojtechsokol) [reply repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/library.py:19]: From [Contribution guidelines](https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md):
> Follow PEP 257 - Docstring Conventions

And from [PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) for One-line Docstrings:
> The closing quotes are on the same line as the opening quotes. This looks better for one-liners.

Comment (Rezney) [reply repos/system_upgrade/el7toel8/actors/ipuworkflowconfig/libraries/library.py:19]: Then I probably misunderstood it in my previous case. Fair enough...

**Comment by vojtechsokol** [comment]
I discussed the issue with actors `OSReleaseCollector` and `CheckOSRelease` with @pirat89 and we aggreed that existing actors will be modified in separate follow-up PR.

**Pull request merged**